ConsoleCalculator/Calculator.cs

```diff
@@ -3,12 +3,258 @@
 namespace ConsoleCalculator
 {
     public class Calculator
-    {
-
-        public string SendKeyPress(char key)
+    {

```

Comment:
Scenario 3 is not working.
Scenario 5 is not working.
Ctrl+C is also not working

Please verify all the scenarios first and then create a pull request for the changes then i will review the pull request completely.